src/locale/ru.json

```diff
@@ -226,6 +226,7 @@
     "Profile": "Профиль",
     "Logout": "Выйти",
     "Network": "Сеть",
+    "NotConnected": "Not Connected",
```

Comment:
for temporary please use google translate

Reply:
Cykaa bylaat ? @irmannmal lol wkkwkw, ok sir

Reply:
resolved in https://github.com/myriadsocial/myriad-web/pull/1542/commits/fae086aeac11935ba33e61be65bdeaae36793dd4 @irmannmal